docs/tests/integration/test_logcollector/test_location/test_exclude.md

```diff
@@ -0,0 +1,25 @@
+# Test location - Exclude
+## Overview 
+
+Check if logcollector works properly with different exclude and locations.

```

Comment:
```suggestion
Check if logcollector works properly with different exclude and location values.

```

Reply:
Fixed in 8e8e84a2a739ed54d49446ac6e5db42ccb2d59dd